RELEASE_NOTES.md

```diff
@@ -1,3 +1,7 @@
+### 6.3.0

```

Comment:
My reasoning for the 6.2.3 numbering was
- there's nothing new to try/use
- it was wrong in the first place
- it's a no brainer no risk change with no external interface change
i.e. someone with a `~> 6.2` spec would not be annoyed or surprised to get it
(I can see the case for 6.3 too and its up to you; just explaining my reasoning)

Reply:
done